.github/workflows/job-test-in-container.yml

```diff
@@ -70,6 +70,12 @@ jobs:
       - name: "Init: expose GitHub Runtime variables for gha"
         uses: crazy-max/ghaction-github-runtime@3cb05d89e1f492524af3d41a1c98c83bc3025124  # v3.1.0
 
+      - name: "Init: install br-netfilter"
+        run: |
+          # This ensures that bridged traffic goes through netfilter
+          sudo apt-get update -qq
+          sudo apt-get install -qq nftables
```

Comment:
How is nftables relevant ?

Reply:
Yeah not relevant, so I have removed it. This was more for debugging to check if the issue was due to having different iptables backend (legacy vs nft)